crates/sp-executor/src/lib.rs

```diff
@@ -44,6 +44,11 @@ mod app {
 /// An executor authority signature.
 pub type ExecutorSignature = app::Signature;
 
+/// An executor authority keypair. Necessarily equivalent to the schnorrkel public key used in
+/// the main executor module. If that ever changes, then this must, too.
+#[cfg(feature = "std")]
```

Comment:
Why only in std? I mean I see you only use it in tests, but what is wrong with exposing it in `no_std`?

Reply:
I copied it from the babe primitive crate(mostly for the comment :D), I don't mind making it available in `no_std`.

Reply:
Okay, the pair is only available under `full_crypto` which I don't think we have to activate https://github.com/paritytech/substrate/blob/fa778cc1187e37bbab132293ddbbb20158139ea5/primitives/application-crypto/Cargo.toml#L37-L40, so I'll keep it in std only.